boa_cli/src/main.rs

```diff
@@ -136,6 +160,22 @@ enum DumpFormat {
     JsonPretty,
 }
 
+#[derive(Debug, Clone, Copy, ValueEnum)]
```

Comment:
I would add some enum-level documentation here. And it might be nice to add links to each format information, and where they can be used, in each variant documentation, or at the enum-level documentation.